src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,149 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.service.RegistrationServiceImpl.MAX_AGE;
+import static core.basesyntax.service.RegistrationServiceImpl.MIN_AGE;
```

Comment:
Better create those values in this file. Not import from the service you are testing